package.json

```diff
@@ -77,6 +77,7 @@
     "numeral": "^2.0.4",
     "react": "^15.4.2",
     "react-relay": "https://github.com/alloy/relay/releases/download/v0.9.3/react-relay-0.9.3.tgz",
+    "request": "^2.81.0",
```

Comment:
Good re-use of dependencies 👍 

I’d still expect a change to the lockfile in most cases. Did you use `yarn add request`?